packages/tocco-ui/src/EditableValue/typeEditors/TimeEdit.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import DurationEdit from './DurationEdit'
+import {calculateMilliseconds} from '../utils'
+
+const TimeEdit = props => {
+  const propHours = (props.value === null || props.value === undefined) ? 0 : props.value.hoursOfDay
+  const propMinutes = (props.value === null || props.value === undefined) ? 0 : props.value.minutesOfHour
```

Comment:
i question the name "propMinutes". why not only minutes, there is no extra value provided by the prop prefix